contracts/prize-strategy/PrizeStrategy.sol

```diff
@@ -367,6 +371,17 @@ contract PrizeStrategy is PrizeStrategyStorage,
     externalErc20s.addAddress(_externalErc20);
   }
 
+  /// @notice Removes an external ERC20 token type as an additional prize that can be awarded
+  /// @dev Only the Prize-Strategy owner/creator can remove external tokens
+  /// @param _externalErc20 The address of an ERC20 token to be removed
+  /// @param _prevExternalErc20 The address of the previous ERC20 token in the `externalErc20s` list.
+  /// If the ERC20 is the first address, then the previous address is the SENTINEL address: 0x0000000000000000000000000000000000000001
+  function removeExternalErc20Award(address _externalErc20, address _prevExternalErc20) external onlyOwner {
+    require(externalErc20s.contains(_externalErc20), "PrizeStrategy/invalid-external-award");
```

Comment:
This condition is already indirectly enforced by this line in MappedSinglyLinkedList#removeAddress:

`require(self.addressMap[prevAddress] == addr, "Invalid prevAddress");`

I *think* we can remove the above require to save gas.  Thoughts?

Reply:
It is enforced already, I agree. But the require statement is misleading in that case since 'addr' could be the invalid value.  I guess we could update the require statement.

Reply:
Nevermind, I see what you did here -- it is assumed that the `addr` is known, and therefore it would be `prevAddress` that is incorrect..